### ISSUE TITLE
JS ERROR: Error: Source.notify() has been moved to Source.showNotification()

### ISSUE THREAD

**Issue by fdinardo** [RaphaelRochet/arch-update#142]
Hi, this is a great extension, thanks for your work.

Running `journalctl /usr/bin/gnome-shell` I see a lot of these errors: 
```
JS ERROR: Error: Source.notify() has been moved to Source.showNotification()this code will break in the future
notify@resource:///org/gnome/shell/ui/messageTray.js:870:23
_showNotification@/home/angelsoul/.local/share/gnome-shell/extensions/arch-update@RaphaelRochet/extension.js:457:21
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
_updateStatus@/home/angelsoul/.local/share/gnome-shell/extensions/arch-update@RaphaelRochet/extension.js:317:11
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
_checkUpdatesEnd@/home/angelsoul/.local/share/gnome-shell/extensions/arch-update@RaphaelRochet/extension.js:431:8
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
_checkUpdatesRead@/home/angelsoul/.local/share/gnome-shell/extensions/arch-update@RaphaelRochet/extension.js:419:8
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
_checkUpdates/this._updateProcess_sourceId<@/home/angelsoul/.local/share/gnome-shell/extensions/arch-update@RaphaelRochet/extension.js:386:97
```

Look like it is not a blocking error, imho it's more like a warning, but it keep pop up all the time, increasing gnome-shell memory leak and logging space.

If you are already aware of it, feel free to close.
I just wanted to let you know.

Thanks.

**Comment by RaphaelRochet** [comment]
Ah, didn't catch this one ! I'll fix it asap. Thanks

**Comment by RaphaelRochet** [comment]
Fixed in 0202312 :)
